Validator/DataCollectingValidator.php

```diff
@@ -16,7 +16,7 @@
 use Symfony\Component\Validator\ConstraintViolationList;
 use Symfony\Component\Validator\ConstraintViolationListInterface;
 use Symfony\Component\Validator\Context\ExecutionContextInterface;
-use Symfony\Component\Validator\Validator\ValidatorInterface;
+use Symfony\Component\Validator\ValidatorInterface;
```

Comment:
isn't this the old interface?

https://github.com/symfony/validator/blob/master/Validator/ValidatorInterface.php